docs/basic-features/image-optimization.md

```diff
@@ -100,7 +100,15 @@ The following describes the caching algorithm for the default [loader](#loader).
 
 Images are optimized dynamically upon request and stored in the `<distDir>/cache/images` directory. The optimized image file will be served for subsequent requests until the expiration is reached. When a request is made that matches a cached but expired file, the cached file is deleted before generating a new optimized image and caching the new file.
 
-The expiration (or rather Max Age) is defined by the upstream server's `Cache-Control` header.
+The expiration (or rather Max Age) is defined by the upstream server's `Cache-Control` header or by adding configuration to `next.config.js`
```

Comment:
This isn't quite true for how its implemented: its actually a mimimum `max-age` so this config won't be used when the upstream value is larger than the config value.

Reply:
I think this paragraph can be left alone and instead you can modify the algorithm defined on line 113 where it says "If `s-maxage` is found"